packages/manager/modules/cloud-connect/src/details/statistics/template.html

```diff
@@ -0,0 +1,96 @@
+                    name="graphPeriod"
+                    class="form-control"
+                    data-ng-disabled="$ctrl.isGraphLoading"
+                    data-ng-change="$ctrl.loadGraphs()"
+                    data-ng-model="$ctrl.graphPeriod"
+                    data-ng-options="'cloud_connect_stats_period_' + period | translate for period in $ctrl.periodOptions"
+                ></select>
+            </div>
+        </div>
+    </form>
+    <div>
+        <oui-header
+            data-heading="{{::'cloud_connect_stats_type_traffic' | translate}}"
+            data-ng-if="$ctrl.displayedGraph === $ctrl.TYPE.TRAFFIC"
+        ></oui-header>
+        <oui-header
+            data-heading="{{::'cloud_connect_stats_type_error' | translate}}"
+            data-ng-if="$ctrl.displayedGraph === $ctrl.TYPE.ERROR"
+        ></oui-header>
+        <oui-header
+            data-heading="{{::'cloud_connect_stats_type_light' | translate}}"
+            data-ng-if="$ctrl.displayedGraph === $ctrl.TYPE.LIGHT"
+        ></oui-header>
+
+        <oui-tile-definition data-ng-if="$ctrl.isLoading">
+            <oui-tile-description>
+                <oui-skeleton data-randomized></oui-skeleton>
+                <oui-skeleton data-randomized></oui-skeleton>
+                <oui-skeleton data-randomized></oui-skeleton>
+                <oui-skeleton data-randomized></oui-skeleton>
+                <oui-skeleton data-randomized></oui-skeleton>
+                <oui-skeleton data-randomized></oui-skeleton>
+                <oui-skeleton data-randomized></oui-skeleton>
+            </oui-tile-description>
+        </oui-tile-definition>
+
+        <div data-ng-if="!$ctrl.isLoading">
+            <div data-ng-if="$ctrl.data.length > 0" class="statistics-graph">
+                <canvas
+                    id="line"
+                    class="chart chart-line"
+                    data-chart-series="$ctrl.series"
+                    data-chart-colors="$ctrl.colors"
+                    data-chart-data="$ctrl.data"
+                    data-chart-labels="$ctrl.labels"
+                    data-chart-options="$ctrl.options"
+                    data-ng-if="$ctrl.displayedGraph === $ctrl.TYPE.TRAFFIC || $ctrl.displayedGraph === $ctrl.TYPE.ERROR"
+                >
+                </canvas>
+                <canvas
+                    id="line"
+                    class="chart chart-line"
+                    data-chart-series="$ctrl.series"
+                    data-chart-colors="$ctrl.colors"
+                    data-chart-data="$ctrl.data"
+                    data-chart-labels="$ctrl.labels"
+                    data-chart-options="$ctrl.lightOptions"
+                    data-ng-if="$ctrl.displayedGraph === $ctrl.TYPE.LIGHT"
+                >
+                </canvas>
+            </div>
+            <div data-ng-if="$ctrl.data.length == 0">
+                <span data-translate="cloud_connect_stats_no_data"></span>
```

Comment:
**nitpick:** Useless `span` element. We can apply `translate` directive directly on the parent `div` element.